stripe/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowControllerInitializer.kt

```diff
@@ -129,6 +130,28 @@ internal class DefaultFlowControllerInitializer(
         )
     }
 
+    private suspend fun setLastSavedPaymentMethod(
+        prefsRepository: PrefsRepository,
+        isGooglePayReady: Boolean,
+        paymentMethods: List<PaymentMethod>
+    ) {
+        if (prefsRepository.getSavedSelection() == SavedSelection.None) {
+            when {
```

Comment:
hmm this check behaves differently than previous changes: 
In previous change, if both `isGooglePayReady` and `paymentMethods.isNotEmpty()` are `true`, I think GooglePay will take precedence we'll set savePaymentSelection to `PaymentSelection.GooglePay` 
however in this case `paymentMethods.first()` will take precedence, we won't set default to GooglePay.
Is this expected?

Reply:
confirmed after offline sync that `paymentMethods.first()` should take precedence